src/main/java/com/epam/controller/ReservationController.java

```diff
@@ -20,20 +20,23 @@ public ReservationController(ReservationService reservationService) {
     }
 
     @GetMapping("/testadmin")
-    public String listReservations(ModelMap modelMap) {
-        List<Reservation> list = reservationService.listReservations();
+    public String testadmin() {
+        return "redirect:/testadmin/1";
+    }
+
+    @GetMapping("/testadmin/{id}")
+    public String listReservations(@PathVariable int id, ModelMap modelMap) {
+        List<Reservation> list = reservationService.listReservations(id, reservationService.totalAmountOfRows);
```

Comment:
I think would be better name it smth like `reservations`

Reply:
done